bin/main.html

```diff
@@ -613,22 +619,22 @@
       <div class="advantage-row">
         <div>Devout</div>
         <input name="attr_advantage_devout_minus" type="checkbox" value="1" hidden="true" />
-        <select name="attr_advantage_devout_count" class="skillexpertise">
+        <select class="skillexpertise" name="attr_advantage_devout_count">
           <option value=" "></option>
           <option value="1">1</option>
         </select>
         <span class="CP" name="attr_advantage_devout_CPdescription"></span>
       </div>
       <fieldset class="repeating_extraadvantage">
-        <!--- We leave no space between the inputs, so there won't be
-              any inside the repeating section -->
-        <input name="attr_extraadvantagename" class="extraadvantagename" type="text" /><input
-          name="attr_extraadvantageCP" class="extraadvantageCP" type="number" /><span
-          class="CP extraadvantageCPdescription" name="attr_extraadvantageCPdescription"></span>
+        <!--
+        - We leave no space between the inputs, so there won't be
+        any inside the repeating section
```

Comment:
This comment isn't right any more, because pug is putting in spaces. So it should go away.